internal/security-governance-system/webapp/src/main/java/org/wso2/security/tools/sgs/webapp/Application.java

```diff
@@ -15,7 +15,8 @@
 * specific language governing permissions and limitations
 * under the License.
 */
-package org.wso2.security.tools.am.webapp;
+
+package org.wso2.security.tools.sgs.webapp;
```

Comment:
```suggestion
package org.wso2.security.tools.scanmanager.web;
```
Let's change other places accordingly.